include/supernode/config_loader.h

```diff
@@ -0,0 +1,21 @@
+
+#pragma once
+
+namespace graft::supernode {
+
+class Config;
+
+class ConfigLoader
+{
+public:
+    ConfigLoader(void);
```

Comment:
It is most definitely related to C and is only allowed in C++ code to allow compatibility with C code.  (But since this is a *constructor* it is *especially* pointless since C compatibility isn't even possible in the first place).  See [the C++ Core Guideline's strong recommendation against this](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#nl25-dont-use-void-as-an-argument-type).